trymito.io/pages/industries/financial-services.tsx

```diff
@@ -0,0 +1,227 @@
+                                </p>
+                            </div>
+                        </div>
+                        <div className={pageStyles.subsection}>
+                            <div className={securityStyles.security_bullet_container}>
+                                <div className={securityStyles.icon}>
+                                    <Image  src={ComputerIcon} alt='icon'></Image>
+                                </div>
+                                <h3>
+                                    Large Data Analysis
+                                </h3>
+                                <p>
+                                    When you get a dump of 5 million rows of data, use Mito to analyze the data yourself instead of waiting for a supporting data scientist to aggregate the data for you.
+                                </p>
+                                <p>
+                                    Even in the highest functioning orgs, waiting for a supporting data scientst adds a full day of overhead to your analysis. 
+                                </p>
+                                <p>
+                                    Getting access to large datasets should increase the quality of your analysis, not decrease it. That requires the subject matter expert to analyze the data.
+                                </p>
+                            </div>
+                            <div className={pageStyles.subsection_second_element_mobile_spacing}>
+                                <div className={securityStyles.icon}>
+                                    <Image className={securityStyles.icon} src={ExcelIcon} alt='icon'></Image>
+                                </div>
+                                <h3>
+                                    Future-proofing Excel Files 
+                                </h3>
+                                <p>
+                                    Teams have gotten away with treating Excel files as databases for the last 10 years. 
+                                    Now these files are on the brink of failure because they hold too much data.
+                                </p>
+                                <p>
+                                    Mito lets your team prep the data using the same Excel process they&apos;ve used for the past 10 years, and push the data directly into a database. 
+                                </p>
+                                <p>
+                                    Not relying on Excel files that need end of life care means your team can focus on decision making.
+                                </p>
+                            </div>
+                        </div>
+                    </section>
+                    <section>
+                        <div className={pageStyles.subsection}>
+                            <h2 className='center'>
```

Comment:
<img width="1535" alt="Screen Shot 2023-07-05 at 2 38 08 PM" src="https://github.com/mito-ds/monorepo/assets/21002431/38c99b42-b707-4dac-a046-aa257410beae">

spacing is strange. can we align top of text with top of image vertically?

Reply:
Or maybe just make image slightly smaller?